index.js

```diff
@@ -47,7 +45,7 @@ function parser (inputs, output, cb) {
         allowHashBang: true,
         ecmaVersion: Infinity,
         sourceType: 'module',
-        plugins: { jsx: true },
+        plugins: ["jsx"],
```

Comment:
Would you mind change this to single quote so it could pass the eslint check from standard.js?